### PR TITLE
lab06 edits

### PR DESCRIPTION
**Pull request by rishab231**
Hi!

The lab looks great. I've made some very minor changes.

### CODE REVIEW

File: notebooks/Lab6.ipynb [hunk 1]
```diff
@@ -204,6 +213,19 @@
...
+   "cell_type": "code",
+   "execution_count": 27,
+   "metadata": {
+    "collapsed": true,
+    "deletable": true,
+    "editable": true
+   },
+   "outputs": [],
+   "source": [
+    "# Run any code you want to for this question\n"
```
Comment (akmorrow13) [inline notebooks/Lab6.ipynb]: Can you fix this sentence? 'Run any code you want for this question'

**Comment by akmorrow13** [comment]
Thanks! Looks good! I put one comment in the PR, if you can address that then I will merge!

**Comment by akmorrow13** [comment]
PING on this. Please complete fixes this as soon as possible.

**Comment by rishab231** [comment]
Hey I have done it!

**Comment by akmorrow13** [comment]
thanks @rishab231 !

**Pull request merged**